### ISSUE TITLE
Why? Unable to connect with your settings

### ISSUE THREAD

**Issue by xiao-k8** [niceit/tabby-cloud-sync-settings#60]
I use Webdav synchronization, and the same settings are used on both computers. One computer works normally, but the other reports an error and cannot be added

Normal computer: Macbook Pro Intel chip
The computer that reported an error: Mac mini M4 chip

我使用Webdav同步的方式，两台电脑相同的设置，一台正常，一台报错无法添加
正常的电脑: Macbook Pro Intel芯片
报错的电脑：Mac mini M4
![Xnip2024-12-30_12-56-10](https://github.com/user-attachments/assets/55ab8dde-251d-4270-bedd-41246e253dc5)
![Xnip2024-12-30_12-55-31](https://github.com/user-attachments/assets/ea4d5df8-e3fa-4134-9383-7ee1ffcbad59)

**Comment by niceit** [comment]
@xiao-k8 you could open (toggle) Tabby application dev tool to inspect the error there

**Comment by xiao-k8** [comment]
> @xiao-k8 you could open (toggle) Tabby application dev tool to inspect the error there

I found the reason. It's because the server returned a 401 status code, which caused the creation of the test.txt file to fail.
I compared the successful machine and it was created successfully

It's strange that I compared two data packets and found that the request header Authorization is the same, as are other request header fields. I don't know why this error occurred. It may be due to my network. I will continue to investigate. Thank you.
